.changeset/forty-goats-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Added new icons “alert-diamond” + “alert-diamond-fill” + “closed-caption” + “info-fill”
```

Comment:
q: When you wrote "regenerated the assets in output", given that it changes the paths of many other icons, should it be mentioned here? Realize it's not a user-facing change, but it's somewhat hard to follow what "regenerated the assets in output" means. 

Looking at this for `info-fill`, it looks a-ok to me otherwise

Reply:
> When you wrote "regenerated the assets in output", given that it changes the paths of many other icons, should it be mentioned here?

It's an issue with Figma. They sometimes change internals of how they generate the SVG paths, and it happens 90% of the times there are some changes. So it would be in every release note about icons. These are no visible changes for the user. And there is no way to test these changes and see if they are breaking somehow:
- manually, is not possible (how do you compare two SVG internal paths?)
- automatically, it could be done with Percy, but testing 1000 icons every time (1000 screenshots) is VERY expensive! (the only solution would be to have a custom tool for visual regression testing; that's what I was using in my prev company, and we tested each asset independently so we could catch single unexpected icon changes).